### ISSUE TITLE
Role Responsibilities

### ISSUE THREAD

**Issue by ghost** [dwyl/hq#324]
We need a stable framework that defines where responsibilities lie in the organisation that will enable us to effectively work as a unit with efficient decision making and autonomy in specialist areas.

**Comment by markwilliamfirth** [comment]
Currently it's unclear who the decision making responsibilities sit with in some areas. For example:

People / HR / Culture
- Which employee benefits should be authorised?
- When, what and how organisation operations and decision making should be made public (remuneration, hiring procedure, company finances)?
- How hiring decisions should be made - who decides when and who to hire

Finance
- Budgets for company activities
- Financial policies e.g. no spending without management accounts
- Which banking provider we bank with

Company Priorities
- Where we should focus our time and resources

Company Website
- Content, IA, focus

We need to effectively delegate these responsibilities.

**Comment by markwilliamfirth** [comment]
Finance
- Director remuneration
- Financial priorities

**Comment by markwilliamfirth** [comment]
+1 Company Priorities - Where we should focus our time and resources

I am still unsure as to who is responsible for this. I assume that currently it is either Inês or a joint voting responsibility of Inês, Nelson and myself.

Either way this would mean that in this case (https://github.com/dwyl/video/issues/29#issuecomment-302686617) this should remain a priority-2 as both Inês and I think it should. Undermining Inês publicly like this does not make for a stable working Director unit.

In summary we need a clear understanding of who sets company priorities.

**Comment by iteles** [comment]
I can answer this last one pretty quickly. 

I set company priorities. Which of course I will discuss at a high level with my fellow 'directors'.
The issue you are referring to is a bad example of this as the change in priority label came with a clear, concise and polite explanation as to why it was a suggested change. There are better ones. 

I agree this needs to be written down and made clear.

**Comment by markwilliamfirth** [comment]
company blog

**Comment by markwilliamfirth** [comment]
@iteles we should review and close this issue as soon as possible - what's a realistic time for when you can get this one done by?

**Comment by markwilliamfirth** [comment]
Who is the PO for https://github.com/dwyl/process-handbook/ ? Will assume me for now

**Comment by iteles** [comment]
To be clear, we are not operating in a vacuum.

We each have job descriptions as a starting point. However, this is a startup
and therefore needs are constantly evolving and taking new shape. This can manifest
as new challenges that we didn't expect to take on (such as FocusHub) or as things
we thought were going to happen imminently but had to wait until other parts of
the organisation found their footing first (such as the time app).
Working in a startup you actually care about means _everyone_ needs to be ready to
take on the jobs that _need_ to be done, whether they 'fit your job description' or not.

For the record, I have noted below our original job descriptions as agreed
in early February 2017 in bullet points. To these, I have added some commentary
on examples and evolutions where relevant in sub-bullet points and as an
introductory paragraph.

These are all governed by a set of principles which we need to formalised (https://github.com/dwyl/hq/issues/407) and importantly, a set of
objectives which were added to the README of this repo 6 months ago https://github.com/dwyl/hq#one-metric-that-matters.

### CEO
In essence I see this role as the person who sets the [vision and direction of
the organisation](https://github.com/dwyl/phase-two#guiding-principles) and is
accountable for when things happen or don't happen. The 'CEO' is responsible for
allocation of people and money (though I think both of these steps can and should
be delegated _to an extent_ once our financial situation is finally clear) and
building relationships with clients and partners.
In my case, I also do a lot of the day to day project work, establishing our
processes and training people to take over this role.
But most importantly, the CEO is the person in a startup who picks up on the things
that need to be done, no matter how big or small and is responsible for making
sure that they either do them themselves (even if its something like [buying a doormat](https://github.com/dwyl/focus-hub/issues/41#issuecomment-2978108830) when
it's raining and no one else is getting it done) or more importantly, ensures
someone trustworthy and with the right skillset is actively working on that
task or group of tasks/area of the business.

Official role description from February 2017:
+ Defining and disseminating the overall vision and direction of the company
+ Determining the business goals deriving from this and associated timelines
+ Setting and modelling company values
+ Business development and relationship management with potential new partners and clients
+ Building and leading the senior team to the point where hiring of staff outside of this team can be delegated
+ Allocating the company’s resources
+ Overseeing financial and fiscal commitments
+ Publicising dwyl and its offerings through personal appearances, talks and
modelled behaviours in the technology community
+ Passing on client, stakeholder and project management skillset to the rest
of the company through shadowing and documentation
+ Evaluating the success of the company and its goals
+ Awareness and dissemination of the internal and external competitive landscape


### COO
There are two key things to remember in this role:
+ This role is summarised as 'to support the CEO in developing the organisation'
and is therefore ever-changing based on the needs of the organisation
  + The idea being that there are a lot of things that need to be done at a
  company that are not just the delivery of service and they are too many for a
  CEO to do alone; having a single view which is uninformed by others is also
  not always a good idea! 
  + Examples of this in our case include: https://github.com/dwyl/hq/issues/363, https://github.com/dwyl/hq/issues/382
+ The key thing for this role is documenting and _automating_ as much of this as
is possible, either to machines or utilising other resources such as virtual
assistants or other team members

Official role description from February 2017:
+ Operational   Process   Implementation: Create   processes   and   operational   plans   for   delivering   the   Company’s   services
  + **[Note from @iteles]** To me, this is the most fluid of the bullet points
  and is essentially a catch-all for 'doing the jobs that need to be done' as  'operational plan' doesn't actually mean anything. A rephrasing could be:  'Operational tasks will be planned, documented and carried out as and when the
  need arises for them in the organisation', where 'operational tasks' refer to
  non-technical, non-strategic/visionary tasks that keep the wheels turning and
  are the backbone of the organisation. This includes a number of
  things described in bullet points below (such as metrics, financials, legals and marketing) but good examples of this also include solving human resources
  issues like what/how/when to provide feedback to dwylers https://github.com/dwyl/hq/issues/182, the dwyler handbook https://github.com/dwyl/hq/issues/242, improving processes https://github.com/dwyl/hq/issues/383 and
  processes for onboarding new joiners https://github.com/dwyl/hq/issues/313
  + It's probably not a great one to start on as it's quite vague, but is a requisite part of the
  role  
+ Operational   Efficiency: Develop and implement strategies for tracking metrics and improving key business
activities,   whilst   increasing   opportunity   for   the   business   to   scale
  + **[Note from @iteles]** Examples of this: financial metrics https://github.com/dwyl/hq/issues/321,
  marketing metrics https://github.com/dwyl/dwyl-site/issues/222 (we have none for social media yet, but should),
  progress towards goals https://github.com/dwyl/focus-hub/issues/101
+ Finances: Work with the Company’s accountants and administrative professionals to manage the Company’s general accounting, payments, invoices, payroll, PAYE, NI, VAT,
cashflow   forecasts,   budgeting,   bookkeeping,   grants,   tax   and   tax   credits
+ Legals: Work with the Company’s lawyers to manage and maintain the Company’s legal agreements and dealings including suitable legal documentation for client contracts, project agreements, employment agreements, contractor agreements, shareholder agreements, trademark applications, filings for Companies House including Confirmation Statements and notifications of appointment and termination of directors and   issuance   of   shares   and   general   corporate   legal   structure
+ HR: Manage the human resources of the company and update and manage the
Employee Handbook and company policies, record and maintain a database of employee details and management of the HR system, ensure the Company has suitable insurance, job description creation, management of the hiring process, onboarding, feedback system and reviews, company culture, team development, team   bonding   and   management   of   OKRs   and   KPI   targets
+ Events: Organise and manage the Company’s internal and external events including invites,
venues, food and drink, event merchandise, audiovisual recording (where
appropriate),   logistics   and   event   post   mortem   reports
+ Office   management: Manage the office environment and inventory, asset management, office supplies and services such as internet access, printing and stationery, management of first aid and fire safety
+ Clients and stakeholders: Manage and maintain relationships with the Company’s  key clients and stakeholders
  + **[Note from @iteles]** This obviously could not happen on day one as we need
  our messaging to be congruent and there was a period of 'learning the ropes'. 
  With one client relationship fully owned by this role and others interacting
  extensively in my absence, this transition is slowly being completed and needs
  to increase as time goes on
+ Branding   and   marketing: Creation and management of branding guidelines, Company logos, fonts, colours and merchandise   designs + Management   of   the   Company’s   frontend   site
  + **[Note from @iteles]** Whereas the establishment of what the site should 
  contain required more input from the group of directors as a whole given it is
  our window to the world, the management of the website project was undertaken 
  as stated
+ PR: Management of PR, press and media
+ Product: Management of internal products
  + **[Note from @iteles]** dwylbot and the website fell into this category, with
  'library' being the only one that was carried out by myself because this was 
  part of the apprenticeship program

### CTO
+ Translate the Company’s vision/mission (defined by CEO+team) into a Technical Roadmap.
+ Set technical vision/direction for the team: https://github.com/dwyl/technology-stack
+ Build the Product(s) that achieve the vision/mission: https://github.com/dwyl/start-here#what
+ Ensure that our technology and product is fit-for-purpose (a joy to use, maintain & scale)
+ Ensure technology choices are captured and clearly communicated. technology-stack/issues
+ Acquire & share technical knowledge https://github.com/search?q=org%3Adwyl+learn
+ Mentor existing and future team members: https://github.com/dwyl/learn-redux/issues/40
+ Define the workflow for the technical team: https://github.com/dwyl/contributing
+ Foster an online/distributed/remote learning culture/community and fuel innovation!
+ Assist/advise other members/partners with technology decisions when requested.

**Comment by iteles** [comment]
> Early stage. [...] It's probably less than 20 people on the team. There's not necessarily a product/market fit. 
> Here it's really great if you're a generalist. If you're enthusiastic about the product and the company and you're _comfortable with extremem ambiguity_. 
> [...] The super-skill here is being the jack of all trades because you actually can be the operations person and the customer service person and the marketing person because there are only five people on your team.

~ Eurie Kim,  Forerunner Ventures

At 34:38 :arrow_right: https://cdnapisec.kaltura.com/p/2279881/sp/227988100/embedIframeJs/uiconf_id/39241941/partner_id/2279881?flashvars[mediaProxy.mediaPlayFrom]=2078&iframeembed=true&playerId=kaltura-player-0_wj8mdnfl&entry_id=0_wj8mdnfl&flashvars[streamerType]=auto
